### PR TITLE
Graduate Chisel

### PR DESCRIPTION
**Pull request by jackkoenig**
Getting the ball rolling, from going over this I think we have the following TODO:

- [x] Write Mission Statement
- [x] Accept LF Code of Conduct
- [x] Adopt CHIPS Alliance IP Policy
- [x] Add link to Chips Alliance on website

### CODE REVIEW

File: projects/chisel.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ primary_contact:
 # Data about active project contributors
 active_committers: https://github.com/chipsalliance/chisel/graphs/contributors
 # Information about project's release methodology and mechanics
-release_methodology: not yet
+release_methodology: As of Chisel 5.0.0 following Semantic Versioning 2.0.0, Chisel3 3.0 - 3.6 followed PVP
```
Comment (jackkoenig) [inline projects/chisel.yml:33]: The release process is fully automated on Github, should I also add information about that?

File: projects/chisel.yml [hunk 2]
```diff
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
 # Did the project put links to CHIPS in the header or footer of its website? If no website, still answer "yes" as declaration of will to do so.
-chips_header_footer_text_on_website: yes
+chips_header_footer_text_on_website: not yet
 # Did the project transfer any registered trademarks and domains to CHIPS? If none are present, still answer "yes" as declaration of intent if that should change.
-trademarks_and_domains_transferred_to_lf: not yet
+trademarks_and_domains_transferred_to_lf: yes
```
Comment (jackkoenig) [inline projects/chisel.yml:45]: I don't think we have any trademarks, and we have transferred chisel-lang.org over.

File: projects/chisel.yml [hunk 3]
```diff
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
```
[the remark this answers is not in the document]
Comment (mgielda) [reply projects/chisel.yml:41]: Adopting the CLA + explicitly confirming it here by saying "yes" is enough.

File: projects/chisel.yml (hunk 2 above)
Comment (mgielda) [reply projects/chisel.yml:45]: Excellent.

File: projects/chisel.yml [hunk 4]
```diff
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
...
+chips_header_footer_text_on_website: not yet
 # Did the project transfer any registered trademarks and domains to CHIPS? If none are present, still answer "yes" as declaration of intent if that should change.
-trademarks_and_domains_transferred_to_lf: not yet
+trademarks_and_domains_transferred_to_lf: yes
 # Describe briefly the project's security vulnerability reporting process, or just defer to https://github.com/chipsalliance/tsc#reporting-security-vulnerabilities
-security_vulnerabilities_reporting_process: not yet
+security_vulnerabilities_reporting_process: Please report via Chisel Github repo: https://github.com/chipsalliance/chisel/security/advisories/new
 # (For specifications only) link to reference implementation of the specification, N/A for non-spec projects
-spec_public_reference_implementation: N/A
+spec_public_reference_implementation: No spec for Chisel, but FIRRTL IR has a spec: https://github.com/chipsalliance/firrtl-spec
```
[the remark this answers is not in the document]
Comment (mgielda) [reply projects/chisel.yml]: Did this. Seems like CI is green.

File: projects/chisel.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jackkoenig) [reply projects/chisel.yml:41]: Sounds good, working on switching the CLA over then I will update this to be "yes".

File: projects/chisel.yml [hunk 5]
```diff
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
 # Did the project put links to CHIPS in the header or footer of its website? If no website, still answer "yes" as declaration of will to do so.
-chips_header_footer_text_on_website: yes
+chips_header_footer_text_on_website: not yet
```
Comment (jackkoenig) [inline projects/chisel.yml:43]:
```suggestion
chips_header_footer_text_on_website: yes
```
Done via the both the logo (which is clickable link) and in the text: https://www.chisel-lang.org

**Comment by jackkoenig** [comment]
I am unable to figure out how to enable the EasyCLA on Chisel repository, and @hcook doesn't seem to have access either. I think it's through this login https://easycla.lfx.linuxfoundation.org/#/?version=2 which I don't have access on the Project/Community management side. Can anyone here enable it, maybe @brianwarner?

**Comment by jackkoenig** [comment]
I've pushed a commit updating some fields, the only thing left is the IP Policy which just requires some help to get EasyCLA enabled on the Chisel repository!

**Comment by kgugala** [comment]
> I've pushed a commit updating some fields, the only thing left is the IP Policy which just requires some help to get EasyCLA enabled on the Chisel repository!

looks like the CLA is enabled there, I see it running in the PRs

**Comment by mgielda** [comment]
It looks like @jackkoenig was able to resolve this with @robmains during our face-to-face (yay!). @jackkoenig can you confirm?

**Comment by jackkoenig** [comment]
The only CLA I see running is Chisel's old CLA--we are trying to enable the LinuxFoundation EasyCLA instead. For example, see https://github.com/chipsalliance/chisel/pull/3417, CLA Assistant is the old one.

I spoke with @robmains at the face-to-face who logged in to the admin interface and it had Chisel marked as "Archived" (even though it clearly isn't) and he didn't have the priviledge to remove that designation so he reached out to the Linux Foundation liaison. Hopefully we hear back soon!

**Comment by mgielda** [comment]
Thanks for the explanation @jackkoenig - seeing the easycla working I somehow associated that with you and Rob figuring it out, but I guess we'll need to wait for LF to fix this.

**Comment by bensternthal** [comment]
@mgielda would you be able to add me as an admin or to an admin team on the chipsalliance org? This would enable me to diagnose easyCLA? Thanks!

File: projects/chisel.yml [hunk 6]
```diff
@@ -30,20 +30,20 @@ primary_contact:
...
 # Link to project's mission statement
-mission_statement_link: N/A
+mission_statement_link: https://github.com/freechipsproject/chisel-lang-governance/blob/407df3f4b02b8be5effc5a39411caacd51d70282/constitution.md#mission-statement
 # Link to logo in .svg format, if present
 svg_logo_link: logos/chisel.svg
 # Did the project accept the LF Code of Conduct? See https://lfprojects.org/policies/code-of-conduct/
-accepted_lf_code_of_conduct: not yet
+accepted_lf_code_of_conduct: yes
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
```
Comment (jackkoenig) [inline projects/chisel.yml:41]:
```suggestion
adopted_chips_alliance_ip_policy: yes
```

**Comment by robmains** [comment]
That's great news! Thanks all for helping get this across the finish line.

Rob
Rob Mains
General Manager
CHIPS Alliance
(c) 1.408.607.4879

**Pull request merged**